UnityProject/Assets/Scripts/Objects/Mining/OreRedemptionMachine.cs

```diff
@@ -114,6 +115,7 @@ public void ClaimLaborPoints(GameObject player)
 				if (itemSlot.ItemObject)
 				{
 					var idCard = AccessRestrictions.GetIDCard(itemSlot.ItemObject);
+					ScoreMachine.AddToScoreInt(laborPoints, "laborPoints");
```

Comment:
might be worth making a Enum / static reference to the "laborPoints" stringy

Reply:
will do

Reply:
done.